pkg/nodeconfig/payload/payload.go

```diff
@@ -140,7 +140,7 @@ $cni_template=$cni_template.Replace("provider_address",$provider_address)
 # Compare CNI config with existing file, and replace if necessary
 $existing_config=""
 if(Test-Path -Path CNI_CONFIG_PATH) {
-    $existing_config= Get-Content -Path "CNI_CONFIG_PATH"
+` + "    $existing_config=((Get-Content -Path \"CNI_CONFIG_PATH\" -Raw) -Replace \"`r\",\"\")" + `
```

Comment:
Interesting to repalce **`r** rather than **\r**. Is this some powershell/string formatting magic?

Reply:
yep, backtick is the escape character in windows/powershell.
As backslash is used for filepaths